[19.023418427] 2018-01-22 22:56:22,758 - INFO - launcher: Starting sanity checks
[19.524335861] 2018-01-22 22:56:22,758 - INFO - launcher: Checking for Python 3.5+
[20.026206970] 2018-01-22 22:56:22,759 - INFO - launcher: Checking console encoding
[20.026206970] 2018-01-22 22:56:22,759 - INFO - launcher: Setting console encoding to UTF-8
[20.026206970] 2018-01-22 22:56:22,759 - INFO - launcher: Ensuring we're in the right environment
[21.027565002] 2018-01-22 22:56:22,760 - INFO - launcher: Adding local bins/ folder to path
[21.529197693] 2018-01-22 22:56:22,760 - INFO - launcher: Required checks passed.
[22.531986237] 2018-01-22 22:56:22,761 - INFO - launcher: Optional checks passed.
[22.531986237] 2018-01-22 22:56:22,761 - INFO - launcher: Moving old musicbot log

######################### PRE-RUN SANITY CHECKS PASSED #########################

[0.8322598934173584] 2018-01-22 22:56:23,571 - DEBUG - musicbot.bot | In bot.py::MainThread(12628), line 213 in _setup_logging: Set logging level to INFO
[0.8322598934173584] 2018-01-22 22:56:23,571 - INFO - musicbot.bot | In bot.py::MainThread(12628), line 80 in __init__: =========== MusicBot (version 1.9.6_2) ===========
[0.8327538967132568] 2018-01-22 22:56:23,571 - INFO - musicbot.bot | In bot.py::MainThread(12628), line 86 in __init__: Loaded autoplaylist with 131 entries
[0.834747314] launcher-INFO: Connecting
[3.6301572322845459] 2018-01-22 22:56:26,369 - DEBUG - musicbot.bot | In bot.py::MainThread(12628), line 1011 in on_ready: Connection established, ready to go.
[3.6301572322845459] 2018-01-22 22:56:26,369 - DEBUG - musicbot.bot | In bot.py::MainThread(12628), line 324 in _cache_app_info: Caching app info
[3.8103489875793457] 2018-01-22 22:56:26,549 - DEBUG - musicbot.bot | In bot.py::MainThread(12628), line 828 in _scheck_ensure_env: Ensuring data folders exist
[3.8153338432312012] 2018-01-22 22:56:26,554 - DEBUG - musicbot.bot | In bot.py::MainThread(12628), line 844 in _scheck_server_permissions: Checking server permissions
[3.8153338432312012] 2018-01-22 22:56:26,554 - DEBUG - musicbot.bot | In bot.py::MainThread(12628), line 848 in _scheck_autoplaylist: Auditing autoplaylist
[3.8158335685729980] 2018-01-22 22:56:26,554 - DEBUG - musicbot.bot | In bot.py::MainThread(12628), line 852 in _scheck_configs: Validating config
[3.8158335685729980] 2018-01-22 22:56:26,554 - DEBUG - musicbot.config | In config.py::MainThread(12628), line 155 in async_validate: Validating options...
[3.8163347244262695] 2018-01-22 22:56:26,555 - DEBUG - musicbot.bot | In bot.py::MainThread(12628), line 855 in _scheck_configs: Validating permissions config
[3.8163347244262695] 2018-01-22 22:56:26,555 - DEBUG - musicbot.permissions | In permissions.py::MainThread(12628), line 59 in async_validate: Validating permissions...
[3.8173377513885498] 2018-01-22 22:56:26,556 - INFO - musicbot.bot | In bot.py::MainThread(12628), line 1022 in on_ready: Connected to Discord!
[3.8188421726226807] 2018-01-22 22:56:26,557 - INFO - musicbot.bot | In bot.py::MainThread(12628), line 1032 in on_ready: Bot:   405055758456324097/Jan DJ II#8469 [BOT]
[3.8203499317169189] 2018-01-22 22:56:26,559 - INFO - musicbot.bot | In bot.py::MainThread(12628), line 1040 in on_ready: Owner: 329605484409782277/CYG4N#2023

[3.8218584060668945] 2018-01-22 22:56:26,560 - INFO - musicbot.bot | In bot.py::MainThread(12628), line 1043 in on_ready: Server List:
[3.8228528499603271] 2018-01-22 22:56:26,561 - INFO - musicbot.bot | In bot.py::MainThread(12628), line 1044 in <listcomp>:  - Rakowy Discord
[3.8248879909515381] 2018-01-22 22:56:26,563 - INFO - musicbot.bot | In bot.py::MainThread(12628), line 1088 in on_ready: Not bound to any text channels
[3.8263611793518066] 2018-01-22 22:56:26,565 - INFO - musicbot.bot | In bot.py::MainThread(12628), line 1114 in on_ready: Not autojoining any voice channels
[3.8288683891296387] 2018-01-22 22:56:26,567 - INFO - musicbot.bot | In bot.py::MainThread(12628), line 1118 in on_ready: Options:
[3.8303728103637695] 2018-01-22 22:56:26,569 - INFO - musicbot.bot | In bot.py::MainThread(12628), line 1120 in on_ready:   Command prefix: !
[3.8313751220703125] 2018-01-22 22:56:26,570 - INFO - musicbot.bot | In bot.py::MainThread(12628), line 1121 in on_ready:   Default volume: 25%
[3.8328793048858643] 2018-01-22 22:56:26,571 - INFO - musicbot.bot | In bot.py::MainThread(12628), line 1123 in on_ready:   Skip threshold: 4 votes or 50%
[3.8344132900238037] 2018-01-22 22:56:26,573 - INFO - musicbot.bot | In bot.py::MainThread(12628), line 1124 in on_ready:   Now Playing @mentions: Disabled
[3.8358871936798096] 2018-01-22 22:56:26,574 - INFO - musicbot.bot | In bot.py::MainThread(12628), line 1125 in on_ready:   Auto-Summon: Enabled
[3.8398983478546143] 2018-01-22 22:56:26,578 - INFO - musicbot.bot | In bot.py::MainThread(12628), line 1126 in on_ready:   Auto-Playlist: Enabled
[3.8414006233215332] 2018-01-22 22:56:26,580 - INFO - musicbot.bot | In bot.py::MainThread(12628), line 1127 in on_ready:   Auto-Pause: Enabled
[3.8429093360900879] 2018-01-22 22:56:26,581 - INFO - musicbot.bot | In bot.py::MainThread(12628), line 1128 in on_ready:   Delete Messages: Enabled
[3.8442673683166504] 2018-01-22 22:56:26,583 - INFO - musicbot.bot | In bot.py::MainThread(12628), line 1130 in on_ready:     Delete Invoking: Disabled
[3.8457825183868408] 2018-01-22 22:56:26,584 - INFO - musicbot.bot | In bot.py::MainThread(12628), line 1131 in on_ready:   Debug Mode: Disabled
[3.8472816944122314] 2018-01-22 22:56:26,586 - INFO - musicbot.bot | In bot.py::MainThread(12628), line 1132 in on_ready:   Downloaded songs will be saved
[3.8507890701293945] 2018-01-22 22:56:26,589 - INFO - musicbot.bot | In bot.py::MainThread(12628), line 258 in _join_startup_channels: Found owner in "ðŸ”¨ Minecraft"
[3.8523240089416504] 2018-01-22 22:56:26,591 - INFO - musicbot.bot | In bot.py::MainThread(12628), line 267 in _join_startup_channels: Attempting to join Rakowy Discord/ðŸ”¨ Minecraft
[3.8537979125976562] 2018-01-22 22:56:26,592 - DEBUG - musicbot.bot | In bot.py::MainThread(12628), line 437 in get_voice_client: Connection attempt 1 to ðŸ”¨ Minecraft
[3.8542990684509277] 2018-01-22 22:56:26,593 - VOICEDEBUG - musicbot.bot | In bot.py::MainThread(12628), line 376 in join_voice_channel: (join_voice_channel) creating futures
[3.8553020954132080] 2018-01-22 22:56:26,594 - VOICEDEBUG - musicbot.bot | In bot.py::MainThread(12628), line 382 in join_voice_channel: (join_voice_channel) setting voice state
[3.8563055992126465] 2018-01-22 22:56:26,595 - VOICEDEBUG - musicbot.bot | In bot.py::MainThread(12628), line 385 in join_voice_channel: (join_voice_channel) waiting for session id
